packages/file-export/fastfetchbot_file_export/video_download.py

```
@@ -0,0 +1,210 @@
+        ydl_opts = {
+            **base_opts,
+            "paths": {"home": download_dir},
+            "format": "m4a/bestaudio/best",
+            "postprocessors": [
+                {"key": "FFmpegExtractAudio", "preferredcodec": "m4a"}
+            ],
+        }
+    else:
+        if video_format is None:
+            if extractor is None:
+                raise ValueError("extractor cannot be None")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)